fluentbit-kafka-fluentd/README.md

```diff
@@ -0,0 +1,143 @@
+# Face to Face 2020/03 - Fluentbit -> Kafka -> Fluentd -> Elasticsearch
+
+## What?
+The purpose of this hackathon item was to investigate a new collector/aggregator solution for Cluster Logging.
+
+## Why?
+Currently Fluentd takes up decent amount of space on the host nodes in order to collect, enrich, and ship logs.
+
+Fluentbit is a slimmer version of Fluentd (created by the same company) and is written in C. It does not have the same amount of plugins as Fluentd, however we felt it was best to just let it pull logs from the nodes and ship them to the aggregator as fast as possible.
```

Comment:
Would it be helpful to compare actual memory footprints of the two fluents? I grabbed fluentd recently from a fresh CLO install:
fluentd
    11 root      20   0 1020.3m 187.9m  50.6m S  11.9  1.6  54:05.22 fluentd